### PR TITLE
Preemptive Priority Scheduling Algorithm added

### PR DESCRIPTION
**Pull request by BamaCharanChhandogi**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by BamaCharanChhandogi** [comment]
Hey! I understand my Problem. But I can not understand what is the difference.

**Comment by siriak** [comment]
There is whitespace before the function declaration

**Comment by BamaCharanChhandogi** [comment]
Do I like this?
image

File: src/main/java/com/thealgorithms/scheduling/PreemptivePriorityScheduling.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    public Process(String name, int arrivalTime, int burstTime, int priority) {
+        this.name = name;
+        this.arrivalTime = arrivalTime;
+        this.burstTime = burstTime;
+        this.priority = priority;
+    }
+}
+
+public class PreemptivePriorityScheduling {
+
```
Comment (siriak) [inline src/main/java/com/thealgorithms/scheduling/PreemptivePriorityScheduling.java:25]:
```suggestion

```

File: src/main/java/com/thealgorithms/scheduling/PreemptivePriorityScheduling.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    public Process(String name, int arrivalTime, int burstTime, int priority) {
+        this.name = name;
+        this.arrivalTime = arrivalTime;
+        this.burstTime = burstTime;
+        this.priority = priority;
+    }
+}
+
+public class PreemptivePriorityScheduling {
+    public static List<String> preemptivePriorityScheduling(List<Process> processes) {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/scheduling/PreemptivePriorityScheduling.java:25]:
```suggestion
    public static List<String> preemptivePriorityScheduling(List<Process> processes) {
```

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**